client/db/bolt/db_test.go

```diff
@@ -706,15 +709,15 @@ func TestOrderFilters(t *testing.T) {
 			filter: &db.OrderFilter{
 				N: orderCount,
 			},
-			expected: []int{4, 5, 3, 2, 1, 0},
+			expected: []int{4, 3, 5, 2, 1, 0},
```

Comment:
active orders now come before archived orders with not filters specified. Is this ever a problem?

Reply:
Fixed with 831a745